Tribler/Test/Core/TorrentChecker/test_torrentchecker_session.py

```diff
@@ -60,6 +62,17 @@ def test_httpsession_failure_reason_in_dict(self):
         session._process_scrape_response(bencode({'failure reason': 'test'}))
         self.assertTrue(session.is_failed)
 
+    def test_httpsession_on_error(self):
+        session = HttpTrackerSession("retracker.local", ("retracker.local", 80), u"/announce?size=33098770156&comment=nana-aida-film-039-s-collection.html&name=Nana+Aida+Film%26%23039%3Bs+Collection+%28Madonna%2C+Attackers%2C+BeFree%29+%5Bcen%5D+%5B2011-2014+%E3%E3.%2C+All+Sex%2C+Married+Woman%2C+Rape%2C+Mother-in-law%2C+Female+Teacher%2C+DVDRip%5D+%5B28+%D4%E8%EB%FC%EC%EE%E2%5D", 5)
```

Comment:
Could you have the test URL simply include `%80` instead of being this.. colourful?